src/topology/connected.lean

```diff
@@ -1547,3 +1547,38 @@ lemma continuous.connected_components_map_continuous {β : Type*} [topological_s
 continuous.connected_components_lift_continuous (continuous_quotient_mk.comp h)
 
 end connected_component_setoid
+
+/-- A set `s` is preconnected if and only if every
+map into `bool` that is continuous on `s` is constant on `s` -/
+
```

Comment:
This blank line should be removed.

Reply:
Done. Also fixed the style errors that linter complained about.